src/internal/clusterstate/v2.6.0/ppsdb.go

```diff
@@ -0,0 +1,34 @@
+package v2_6_0
+
+import (
+	"context"
+
+	"github.com/pachyderm/pachyderm/v2/src/pps"
+
+	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
+	"github.com/pachyderm/pachyderm/v2/src/internal/pachsql"
+)
+
+func migrateBranchlessCommitsPPS(ctx context.Context, tx *pachsql.Tx) error {
+	jis, err := listCollectionProtos(ctx, tx, "jobs", &pps.JobInfo{})
+	if err != nil {
+		return errors.Wrap(err, "collecting jobs")
+	}
+	for _, ji := range jis {
+		ji.OutputCommit.Repo = ji.OutputCommit.Branch.Repo
```

Comment:
I think we should consider niling out the branch field. It looks like we are still setting it when creating a commit, but that doesn't seem to fit the model we are pushing for (i.e. branches are just pointers to commits). Keeping the branch field in the proto for API compatibility makes sense, but storing it doesn't seem like the right move even if you could imagine some potential value in knowing what branch was used when creating the commit.